Altis_Life.Altis/stringtable.xml

```diff
@@ -7828,7 +7828,7 @@
             <Spanish>Esta persona ya reapareció o fue revivida.</Spanish>
             <Russian></Russian>
             <German>Diese Person ist entweder respawnt oder wurde bereits wiederbelebt.</German>
-            <French>Cette personne à soit réapparue, soit a déjà été réanimée.</French>
+            <French>Cette personne a peut etre fait une réapparition ou a déjà été réanimée.</French>
```

Comment:
Without dash (-) : "peut être" = "can be"
Exact should be "peut-être" = "perhaps/maybe"